src/test/java/api/ItemsByInstanceResourceTest.java

```diff
@@ -77,6 +78,17 @@ void canGetInstanceById() {
       hasJsonPath("tenantId", is(TENANT_ID_UNIVERSITY)))));
   }
 
+  @Test
+  void canGetEmptyResult() {
+    UUID instanceId = UUID.randomUUID();
+
+    ResourceClient.forSearchClient().replace(instanceId, new JsonObject());
```

Comment:
I think we can safely remove this line as well

Reply:
I do not think we can. If we remove this line the below request will fail with 404 status code.